example/llama-html/index.html

```diff
@@ -0,0 +1,647 @@
+<!DOCTYPE html>
```

Comment:
Move this "example" to a separate private repo until we decide what to do with it. I don't think this should be a part of acord examples. It might end up as an acord app, but still separate